speed_python/templates/about.html

```diff
@@ -68,12 +68,12 @@ <h4>The Hosting</h4>
 <h4>Resources</h4>
 <ul>
     <li>
-        <a href="http://mail.python.org/mailman/listinfo/speed" target="_blank">The mailing list</a>
+        <a href="https://mail.python.org/mailman/listinfo/speed" target="_blank">The mailing list</a>
         is here - if you want to volunteer to help out with the project, this
         is the place to start.
     </li>
     <li>
-        <a href="http://mail.python.org/pipermail/speed/" target="_blank">The list archives</a>
+        <a href="https://mail.python.org/pipermail/speed/" target="_blank">The list archives</a>
```

Comment:
Archives moved at: https://mail.python.org/archives/list/speed@python.org/

Reply:
Is https://mail.python.org/archives/list/speed@python.org/latest perhaps a better link? That URL shows the full archive browser, whereas without `/latest` (confusingly) you only see a few "Recent" posts.

Reply:
Is https://mail.python.org/archives/list/speed@python.org/latest perhaps a better link? That shows the full archive list, whereas without the `/latest` you (confusingly) see only a few "Recent" posts by default.